.github/dependabot.yml

```diff
@@ -116,14 +116,28 @@ updates:
     schedule:
       interval: daily
   - package-ecosystem: docker
-    directory: /docker-compose/elasticsearch
+    directory: /docker-compose/elasticsearch/v7
```

Comment:
there is a plural version of `directories`, this can minimize the copy&paste 

https://github.com/dependabot/dependabot-core/blob/1aac2b839744c809d52043fa51f447d71fe9ca48/.github/dependabot.yml#L34C5-L36C31

Reply:
Oh nice. Will make the change

Reply:
can we apply it to other entries too? Most of the `docker` targets above can be collapsed into one

Reply:
Yes. Just saw that. Is it okay if its in the same PR ?

Reply:
yeah